Update power roll messages to show formula per target

cdverrett94 opened the pull request:
Related to #416

- Per that issue, banes/edges should show in the formula, which requires showing the formula per target. I set it up so that the formula and roll total are on the same line to conserve space when there are many targets.
- I didn't change the `2 Edges` to `Double Edge` as noted in the issue.
- In the process, I noticed banes weren't being applied due to a misplaced closing parenthesis.
- Flavor was also being incorrectly set so I fixed that as well


![power-roll-formula](https://github.com/user-attachments/assets/614e6a33-1e61-4cd5-901d-c85ce6f2a01a)

krbz999 commented:
I suggest dropping the flavor on the individual modifiers and implementing a tooltip for the roll for a more detailed overview.

I.e., show "2d10 + 2 + 2 + 2" but on hover show, "2d10 + 2 + 2[Edge] + 2[Bonus]"

krbz999 commented:
> That's a good suggestion! Is there an easy way to strip the flavor from the formula?

```js
for (const term of roll.terms) term.options.flavor = "";
```
Though note that this mutates the evaluated roll, so you may wish to save the original formula for the tooltip.

cdverrett94 commented:
Thanks for that snippet. I've updated per the request and updated the screenshot in the initial post to showcase.

Instead of doing it and mutating the original formula, I added a getter to generate a new roll and stripped the flavor from that roll instead.

JPMeehan commented on src/module/rolls/power.mjs:
```diff
@@ -261,6 +265,16 @@ export class PowerRoll extends DSRoll {
+   * @returns {string}
+   */
+  get flavorlessFormula() {
+    const flavorlessRoll = new this.constructor(this.formula);
```
Might be worth checking on the difference between this function and using `this.clone()`, which uses `new this.constructor(this._formula, this.data, this.options)`

JPMeehan replied:
I think this is fine, your reasoning makes sense to me. Could you add a comment?

cdverrett94 replied:
Done!

JPMeehan reviewed (approved):
Generally looks good, and yeah the tooltip was a good adjustment
